### ISSUE TITLE
Feature request - Add Cemu & Yuzu into consoles automated game management

### ISSUE THREAD

**Issue by T13nou** [ChimeraOS/chimeraos#545]
Hi folks !

Now that both Yuzu & Cemu are available as Flatpak & AppImage, it would be killer to have them integrated into ChimeraOS as native emulators.

Thanks :)

Etienne

**Comment by T13nou** [comment]
Yes this is exactly the guide I'm using but it's pain in the ass to modify manually the yaml files to allow running non present emulators.

This why I'm not declaring this on as a bug but really improvement request :)

**Comment by npaladin2000** [comment]
There's already a launcher for the switch flatpak, looks like it was added a month ago, so all you have to do is add it to the Chimera web app and install the yuzu flatpak. 

https://github.com/ChimeraOS/chimera/blob/master/launchers/switch
